atomic_reactor/plugins/post_tag_by_labels.py

```diff
@@ -66,6 +66,12 @@ def get_label(label_name):
         version = get_label("version")
         release = get_label("release")
 
+        # docker tags cannot be longer than 128 chars
+        # see https://github.com/docker/docker/pull/8447
+        if len(release) > 128:
+            self.log.warn('Release is too long, docker only supports 128 char tags')
+            release = release[:127]
```

Comment:
Ah, good catch! Updated the code and the test.

>we should just fail if we get here

Its probably worth doing in osbs-client or k-cb then, but this should be confirmed. This happened during throughput testing with incorrect parameters and didn't actually occur on prod or any real image